### PR TITLE
Added mmctl options for bulk import/export tasks

### PR DESCRIPTION
**Pull request by cwarnermm**
Added mmctl equivalent commands and processes for bulk import/export operations.

### CODE REVIEW

File: source/manage/bulk-export-data.rst [hunk 1]
```diff
@@ -3,6 +3,34 @@
 Bulk export data
 ----------------
 
+.. tabs::
+
+  .. tab:: Use mmctl
+
+    1. Create a full export file including attachments by running the mmctl export create -- attachments command. See the `Mattermost workspace migration <https://docs.mattermost.com/manage/cloud-data-export.html#create-the-export>`__ documentation for details.
+    
+    2. While the job is running, you can check job status by running the `mmctl export job show <https://docs.mattermost.com/manage/mmctl-command-line-tool.html#mmctl-export-job-show>`__ command.
```
Comment (justinegeffen) [inline source/manage/bulk-export-data.rst:12]:
```suggestion
    2. While the job is running, you can check its status by running the `mmctl export job show <https://docs.mattermost.com/manage/mmctl-command-line-tool.html#mmctl-export-job-show>`__ command.
```

File: source/manage/bulk-export-data.rst [hunk 2]
```diff
@@ -3,6 +3,34 @@
...
+
+    3. When the export job status is successful
+    
+      a. Idenify the name of the completed export file by running the `mmctl export list <https://docs.mattermost.com/manage/mmctl-command-line-tool.html#mmctl-export-list>`__ command.
```
Comment (justinegeffen) [inline source/manage/bulk-export-data.rst:16]:
```suggestion
      a. Identify the name of the completed export file by running the `mmctl export list <https://docs.mattermost.com/manage/mmctl-command-line-tool.html#mmctl-export-list>`__ command.
```

File: source/manage/bulk-export-data.rst [hunk 3]
```diff
@@ -3,6 +3,34 @@
...
+    1.  Navigate to the directory where the Mattermost server is installed. On a default install of Mattermost, the directory is ``/opt/mattermost``.
+    2.  Run the following command to extract data from all teams on the server. Note that you can change the file name and specify an absolute or relative path to dictate where the file is exported:
+  
+        ``sudo -u mattermost bin/mattermost export bulk file.json --all-teams``
+
+        ``sudo -u mattermost bin/mattermost export bulk /home/user/bulk_data.json --all-teams``
+  
+    3.  Retrieve your file from the location you specified.
+
 At this time, the export supports attributes of the objects listed below. All Mattermost Bulk Export data files will begin with a `Version` object as the first line of the file. This indicates the version of the Mattermost Bulk Import file format with which the exported data is compatible.
```
Comment (justinegeffen) [inline source/manage/bulk-export-data.rst:34]:
```suggestion
At this time, the export supports attributes of the objects listed below. All Mattermost bulk export data files will begin with a `Version` object as the first line of the file. This indicates the version of the Mattermost bulk import file format with which the exported data is compatible.
```

File: source/manage/bulk-export-tool.rst [hunk 4]
```diff
@@ -28,20 +28,4 @@ You can export the following data types:
 - Posts (Posts in the Public/Private channels and also replies to those posts)
```
Comment (justinegeffen) [inline source/manage/bulk-export-tool.rst:28]:
```suggestion
- Posts (Posts in the public/private channels and also replies to those posts)
```

**Review by justinegeffen — APPROVED** [review]
Few minor things but otherwise LGTM! :)

**Review by nab-77 — COMMENTED** [review]
as discussed we need to go back around the cli to mmctl change, and then sunset cli use when all features/functions are present in mmctl.

**Comment by cwarnermm** [comment]
Thanks, @nab-77! I'll get a ticket on our docs backlog for this work.

**Pull request merged**